### PR TITLE
add french_healthcard, driver_license, payslip_fra v3

### PR DESCRIPTION
**Pull request by ianardee**

### CODE REVIEW

File: products/payslip_fra/response_v3/complete.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        },
+        "employer": {
+          "address": "1 RUE DU TONNOT 25210 DOUBS",
+          "company_id": "12345678901234",
+          "company_site": null,
+          "naf_code": "1234A",
+          "name": "DEMO COMPANY",
+          "phone_number": null,
+          "urssaf_number": null
+        },
```
Comment (sebastianMindee) [inline products/payslip_fra/response_v3/complete.json:51]: A lot of the fields are missing here, can you fill them in for the tests, please? Same thing for summary_full.rst obviously

File: products/driver_license/response_v1/complete.json [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        "mrz": {
+          "page_id": 0,
+          "polygon": [],
+          "value": null
+        },
+        "place_of_birth": {
+          "page_id": 0,
+          "polygon": [],
+          "value": null
+        },
```
Comment (sebastianMindee) [inline products/driver_license/response_v1/complete.json:241]: Can you give these a value for the tests, please? same thing for summary_full.rst

**Review by sebastianMindee — CHANGES_REQUESTED** [review]
Add more fields on complete.json samples, especially for objects, if possible

File: products/payslip_fra/response_v3/complete.json (hunk 1 above)
Comment (ianardee) [reply products/payslip_fra/response_v3/complete.json]: done

File: products/driver_license/response_v1/complete.json (hunk 2 above)
Comment (ianardee) [reply products/driver_license/response_v1/complete.json]: done

**Review by sebastianMindee — APPROVED** [review]
Looks good

**Pull request merged**